package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "dex-react",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "NODE_ENV=development webpack-dashboard -- webpack-dev-server --mode development --hot",
+    "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
+    "eslint": "eslint . --ext .ts,.tsx,.js",
+    "eslint-fix": "eslint --fix . --ext .ts,.tsx,.js",
+    "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@hot-loader/react-dom": "^16.8.6",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-hot-loader": "^4.12.10",
+    "react-router-dom": "^5.0.1",
+    "styled-components": "^4.3.2",
+    "web3": "^2.0.0-alpha"
```

Comment:
@anxolin What the decision about web3, we did you go for 2.0.0-alpha and not for example1.0.0-beta.55?

Reply:
We should take a decision here.

Any pros/cons?

Reply:
`beta.55 == 2.0.0-alpha` just renamed

Reply:
![image](https://user-images.githubusercontent.com/2352112/62867580-e2496580-bd13-11e9-8d4e-e8439b8f7877.png)

I didn't know, a bit confusing web3, here is some info https://medium.com/@samuel_91690/1-0-release-web3-js-ddd23d3c8f62